src/hardware/gus.cpp

```diff
@@ -1713,7 +1707,9 @@ void GUS_AddConfigSection(Config* conf)
 
 	constexpr auto changeable_at_runtime = true;
 
-	Section_prop* sec = conf->AddSection_prop("gus", &gus_init, changeable_at_runtime);
+	Section_prop* sec = conf->AddSection_prop("gus",
+	                                          &toggle_gus,
```

Comment:
Isn't the name `toggle_gus` a bit misleading? I think I preferred `init_gus`, that makes it clearer this is really just the init function for the GUS section.

I guess "toggling" to me implies only turning something on or off, but `toggle_gus` does more than that: it gets all the GUS settings from the config, configures the GUS, etc. Especially then `toggle_gus` is called for the first time, it initialised the GUS. Then if I only change the DMA and IRQ settings, it doesn't _really_ toggle it, does it? Rather it reconfigures the GUS (which technically involved turning it off then back on, but still 😄 ).

Reply:
Yup! this will be replaced (or has been, maybe in the latest push) with the new `MODULE_Configure(ModuleLifecycle::Create / Reconfigure / Destroy)` approach.